### PR TITLE
Elro K1 connector is not connecting to HA #17

### PR DESCRIPTION
**Pull request by AmpioRosso**
Show exception information in fire_alarm_demo
Check command result for ST_answer_OK message and continue before checking if the result is valid JSON

### CODE REVIEW

File: examples/fire_alarm_demo.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ class AlarmDemo(K1):
 
     async def async_demo1(self) -> None:
         """Main routine to demonstrate the API code."""
+        #logging.basicConfig(level=logging.DEBUG)
```
Comment (jbouwh) [inline examples/fire_alarm_demo.py:28]: I suppose this line can be removed?

File: examples/fire_alarm_demo.py [hunk 2]
```diff
@@ -38,7 +39,8 @@ async def async_demo1(self) -> None:
                     print(
                         f"{key} ({item.get('name')}): status={item['device_state']} data={item['device_status_data']}"
                     )
-            except Exception:  # pylint: disable=broad-except
+            except Exception as e:  # pylint: disable=broad-except
+                print(f"Exception caught {e.args}")
                 pass
```
Comment (jbouwh) [inline examples/fire_alarm_demo.py:44]: Please rename `e` to err and remove the `pass` statement

File: elro/api.py [hunk 3]
```diff
@@ -240,6 +240,8 @@ async def async_process_command(
                         else None,
                     )
                     self._protocol.datagram_data = self._loop.create_future()
+                    if (raw_data[0].decode("utf-8").strip().casefold() == "{ST_answer_OK}".casefold()):
+                        continue
```
Comment (jbouwh) [inline elro/api.py:244]:
```suggestion
                    if (
                        raw_data[0].decode("utf-8").strip().casefold()
                        == "{ST_answer_OK}".casefold()
                    ):
                        continue
```

When does this case happen?

**Review by jbouwh — CHANGES_REQUESTED** [review]
Thanx for the PR.
Left some comments.

File: examples/fire_alarm_demo.py (hunk 1 above)
Comment (jbouwh) [reply examples/fire_alarm_demo.py:28]:
```suggestion
```

Comment (AmpioRosso) [reply examples/fire_alarm_demo.py:28]: Yes, added and commented it for newbies like myself; it gave me a bit more insights while debugging

File: examples/fire_alarm_demo.py (hunk 2 above)
Comment (AmpioRosso) [reply examples/fire_alarm_demo.py]: Done

**Review by jbouwh — APPROVED** [review]
I could not actually reproduce a case where this is needed, but it does no harm as far as I can see.

Thnx @AmpioRosso 👍

**Comment by jbouwh** [comment]
Version 0.1.14 of ha-elro-connects was released that contains this fix. It should be available for download via HACS.

**Comment by AmpioRosso** [comment]
@jbouwh Hi Jan, I installed the latest release and the integration is working for me now, I can see the fire alarms and the wall plug I set up. I ran into two errors before, the one mentioning the missing quotes and an empty one, haven't seen them reappear after installing the latest release.

**Comment by jbouwh** [comment]
That is very good news. BTW Just released an update to support are and entity name translations.

**Pull request merged**